workflows/utils/push.py

```diff
@@ -12,7 +12,7 @@ class Push(luigi.Task):
     input_path = luigi.Parameter()
     output_path = luigi.Parameter()
     parallelism = luigi.IntParameter(default=os.cpu_count(), significant=False)
-    dynamic_requires = luigi.Parameter(
+    dynamic_requires = luigi.ListParameter(
```

Comment:
This is a little suspicious to me, I've had issues with this as a ListParameter in the past. If this empirically works though I have no issues with it,